test/chromedriver-specs.js

```diff
@@ -6,6 +6,8 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import B from 'bluebird';
 import { exec } from 'teen_process';
+import ADB from 'appium-adb';
```

Comment:
These changes to the test are now unnecessary. And this is what is causing CI to fail, since Travis doesn't give access to the Android SDK on Mac.

Reply:
this applies to the `package.json` addition, too. Sorry about that.